cuda_core/docs/source/release/0.3.1-notes.rst

```diff
@@ -0,0 +1,44 @@
+.. SPDX-License-Identifier: Apache-2.0
+
+.. currentmodule:: cuda.core.experimental
+
+``cuda.core`` 0.3.1 Release Notes
+=================================
+
+Released on July 2, 2025
+
+
+Highlights
+----------
+
+- Add a :doc:`Getting Started <getting-started>` page.
+- :class:`Stream` and :class:`Event` creation and some operations are made faster.
+
+
+Breaking Changes
+----------------
+
+None.
+
+
+New features
+------------
+
+None.
+
+
+New examples
+------------
+
+- Add a `CUDA graph <https://github.com/NVIDIA/cuda-python/blob/main/cuda_core/examples/cuda_graphs.py>`_ example.
+- Add a `memory resource <https://github.com/NVIDIA/cuda-python/blob/main/cuda_core/examples/memory_ops.py>`_ example.
+
+
+Fixes and enhancements
+----------------------
+
+- Fix installing optional dependencies ``cuda-core[cu11]`` and ``cuda-core[cu12]`` not including all needed packages.
+- :class:`Buffer` allocated from a :class:`LegacyPinnedMemoryResource` can be passed as a kernel launch argument.
+- Fix the handling for insufficient driver versions when querying :attr:`Kernel.num_arguments` and `:attr:`Kernel.arguments_info`.
```

Comment:
```suggestion
- Fix the handling for insufficient driver versions when querying :attr:`Kernel.num_arguments` and :attr:`Kernel.arguments_info`.
```

Reply:
Approved, but this is needed before merge